releases/release-1.31/release-notes/release-notes-draft.json

```diff
@@ -3980,10 +4370,49 @@
     "duplicate_kind": true,
     "is_mapped": true
   },
+  "125163": {
+    "commit": "f2428d66cc2256fcbe2a486b1403603530f46c6f",
+    "text": "DRA: The Dynamic Resource Allocation (DRA) driver's daemonset must be deployed with a service account that enables writing ResourceSlice and reading ResourceClaim objects.",
+    "markdown": "DRA: The Dynamic Resource Allocation (DRA) driver's daemonset must be deployed with a service account that enables writing ResourceSlice and reading ResourceClaim objects. ([#125163](https://github.com/kubernetes/kubernetes/pull/125163), [@pohly](https://github.com/pohly)) [SIG Auth, Node and Testing]",
+    "documentation": [
+      {
+        "description": "[KEP]",
+        "url": "https://github.com/kubernetes/enhancements/issues/4381",
+        "type": "KEP"
+      },
+      {
+        "description": "[Docs]",
+        "url": "https://github.com/kubernetes/enhancements/pull/4667",
+        "type": "KEP"
+      }
+    ],
+    "author": "pohly",
+    "author_url": "https://github.com/pohly",
+    "pr_url": "https://github.com/kubernetes/kubernetes/pull/125163",
+    "pr_number": 125163,
+    "areas": [
+      "test",
+      "kubelet",
+      "e2e-test-framework"
+    ],
+    "kinds": [
+      "api-change",
+      "feature"
+    ],
+    "sigs": [
+      "auth",
+      "node",
+      "testing"
+    ],
+    "feature": true,
+    "duplicate": true,
+    "duplicate_kind": true,
+    "is_mapped": true
+  },
   "125168": {
     "commit": "d8b33e32b16a8e393f9e174af92b19302c729a98",
-    "text": "job-controller: the `JobReadyPods` feature flag has been removed (deprecated since v1.31)",
-    "markdown": "Job-controller: the `JobReadyPods` feature flag has been removed (deprecated since v1.31) ([#125168](https://github.com/kubernetes/kubernetes/pull/125168), [@kaisoz](https://github.com/kaisoz)) [SIG Apps]",
+    "text": "job-controller: the `JobReadyPods` feature flag has been removed (deprecated since v1.31).",
+    "markdown": "Job-controller: the `JobReadyPods` feature flag has been removed (deprecated since v1.31). ([#125168](https://github.com/kubernetes/kubernetes/pull/125168), [@kaisoz](https://github.com/kaisoz))",
```

Comment:
```suggestion
    "markdown": "Job-controller: the `JobReadyPods` feature flag has been removed (deprecated since v1.31). ([#125168](https://github.com/kubernetes/kubernetes/pull/125168), [@kaisoz](https://github.com/kaisoz)) [SIG Apps]",
```